res/openvpn-gui-res-ru.rc

```
@@ -535,34 +535,34 @@ OpenVPN, возможно, не установлен."
     IDS_ERR_IMPORT_FAILED "Ошибка импорта файла. Путь %ls не может быть создан.\n\n\
 Убедитесь, что у вас есть необходимые привилегии для данного действия."
     IDS_NFO_IMPORT_SUCCESS "Импорт файла успешно завершен."
-    IDS_NFO_IMPORT_OVERWRITE "A config named ""%ls"" already exists. Do you want to replace it?"
-    IDS_NFO_IMPORT_SOURCE "Do you want to import the profile <%ls>?"
-    IDS_ERR_IMPORT_SOURCE "Cannot import file <%ls> as it is already in the global or local config directory"
-    IDS_ERR_IMPORT_ACCESS "Cannot import <%ls> as it is missing or not readable"
+    IDS_NFO_IMPORT_OVERWRITE "Файл конфигурации с именем ""%ls"" уже существует. Вы хотите заменить его?"
+    IDS_NFO_IMPORT_SOURCE "Вы хотите импортировать профиль <%ls>?"
+    IDS_ERR_IMPORT_SOURCE "Нельзя импортировать файл <%ls>, поскольку он уже в папке глобальных или локальных настроек"
```

подумалось - а что такого, что файл уже есть. вроде бы не ошибка по своей сути. после операции импорта (которую можно считать успешной) целевой файл появился.

может перестать считать это ошибкой ?))

---

Yeah but not counting it as an error (importing profile which is already imported) goes beyond the scope of updating Russian localization. We could change the code and then change/remove this line.

---

> Yeah but not counting it as an error (importing profile which is already imported) goes beyond the scope of updating Russian localization. We could change the code and then change/remove this line.

This happens only if the user points to a file already in the config-directory as the source to be imported. An unlikely corner case? But, we could either silently ignore such actions (implicit success) or show success message saying "Already imported" if that's preferred.